e2e/tests/gatekeeperAdministersCaseAfterGatekeeping_test.js

```diff
@@ -60,7 +61,35 @@ Scenario('Gatekeeper enters allocation decision', async (I, caseViewPage, enterA
   I.seeEventSubmissionConfirmation(config.applicationActions.enterAllocationDecision);
 });
 
-Scenario('Gatekeeper enters hearing details and submits', async (I, caseViewPage, addHearingBookingDetailsEventPage) => {
+Scenario('Gatekeeper manages hearings', async (I, caseViewPage, manageHearingsEventPage) => {
+  await caseViewPage.goToNewActions(config.administrationActions.manageHearings);
+  await manageHearingsEventPage.enterHearingDetails(hearingDetails[0]);
+  await manageHearingsEventPage.enterVenue(hearingDetails[0]);
+  await I.retryUntilExists(() => I.click('Continue'), '#judgeAndLegalAdvisor_judgeTitle');
+  await manageHearingsEventPage.enterJudgeAndLegalAdvisorDetails(hearingDetails[0]);
+  await I.retryUntilExists(() => I.click('Continue'), '#sendNoticeOfHearing');
+  await manageHearingsEventPage.sendNoticeOfHearingWithNotes(hearingDetails[0].additionalNotes);
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.administrationActions.manageHearings);
+  caseViewPage.selectTab(caseViewPage.tabs.hearings);
+
+  I.wait(10);
```

Comment:
Still required in only one pass? We could have this flow select the no option for sending notice of proceedings then you wouldn't have the issue.

Reply:
Issue arises because first hearing triggers populate SDO dates, which affects case data, which will cause case data modified outside transaction error